cli/pcluster/config/validators.py

```diff
@@ -421,7 +421,7 @@ def ec2_instance_type_validator(param_key, param_value, pcluster_config):
     warnings = []
 
     try:
-        if param_value not in list_ec2_instance_types():
+        if param_value not in get_supported_instance_types():
             errors.append(
                 "The instance type '{0}' used for the '{1}' parameter is not supported by EC2.".format(
```

Comment:
this probably should become: it's not supported by AWS ParallelCluster

Reply:
Fixed, thanks